sop/modules/OSD/partials/ocs_extend_cluster.adoc

```diff
@@ -0,0 +1,27 @@
+
+To Extend the cluster , and reset is back, follow the below mentioned steps
+
+*Extend the ceph cluster by 0.02*
+
+To extend the ceph cluster, you can now just run the following command 
+
+----
+oc process -n openshift-storage ocs-extend-cluster | oc create -f -
+----
+
+Now the ceph cluster is extended by 0.02, the cluster state would become healthy
+Notify the customer once the cluster is healthy. When the customer is done deleting data ,
+you need to return the threshold to the previous value.
+ 
+but before you reset your cluster back to the previous threshold value, you should delete the existing job resource:
```

Comment:
```suggestion
Now the ceph cluster threshold is extended from 0.85 to 0.87, this will make the cluster to get out of HEALTH_ERR state.
Notify the customer once it is out of HEALTH_ERR state to delete some of the existing data. When the customer is done deleting data ,
you need to revert the threshold to the previous value.
 
Before reverting the threshold, the existing job resource should be deleted:
```